opm/parser/eclipse/Generator/KeywordLoader.cpp

```diff
@@ -109,7 +109,12 @@ namespace Opm {
         std::shared_ptr<ParserKeyword> parserKeyword = std::make_shared<ParserKeyword>(*jsonConfig);
         {
             boost::filesystem::path abs_path = boost::filesystem::absolute( path );
-            addKeyword( parserKeyword , abs_path.string() );
```

Comment:
Then this can't be merged. We should strive to avoid these kinds of \#ifdefs, and rather abstract them properly (i.e. this should either use `generic_string` unconditionally or `addKeyword` should take the boost path and figure this out for itself.

It won't be merged as-is.